src/components/Teaser/DefaultBody.jsx

```diff
@@ -59,11 +59,7 @@ const TeaserDefaultTemplate = (props) => {
             <div className="grid-teaser-item default">
               {(href.hasPreviewImage || href.image_field || image) && (
                 <div className="grid-image-wrapper">
-                  <Image
-                    src={hasImageComponent ? href : defaultImageSrc}
-                    alt=""
-                    loading="lazy"
-                  />
```

Comment:
@iFlameing @sneridagh This is the wrong fix. The problem is that we were never using `image` when there is a custom Image component. Using `defaultImageSrc` in that case is a bad solution, because it prevents the custom image component from getting the full object it needs to render a srcset.

I opened a new PR with my solution: https://github.com/kitconcept/volto-blocks-grid/pull/91